vnext/ReactUWP/packages.config

```diff
@@ -1,5 +1,7 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="boost" version="1.68.0.0" targetFramework="native" />
   <package id="boost_date_time-vc141" version="1.68.0.0" targetFramework="native" />
+  <package id="ReactNative.Hermes.Windows" version="0.1.1" targetFramework="native" Condition="'$(OSS_RN)' != 'true' AND '$(USE_HERMES)' == 'true'" />
+  <package id="ReactNative.V8JSI.Windows" version="0.1.2" targetFramework="native" Condition="'$(OSS_RN)' != 'true' AND '$(USE_V8)' == 'true'"/>
 </packages>
```

Comment:
These references are probably not needed since all the code lives in ReactWindowsCore

Reply:
This is still needed as the ReactWIndowsCOre only produces a static library .. The v8jsi needs to be still linked into the final dlls.. ie. ReactUWP and REactWindowsDesktop ..

Reply:
Are you sure the references are not transitive? Hermes builds fine without this and the vcxproj reference. It could be a matter of how RestoreProjectStyle is defined and/or which version of VS / MSBuild you're using.

---
In reply to: [315905480](https://github.com/microsoft/react-native-windows/pull/2960#discussion_r315905480) [](ancestors = 315905480)

Reply:
Thanks.. I'll check it out Tudor ..